main.js

```diff
@@ -34,20 +34,11 @@ module.exports = {
 	},
 	resolve: function(path, modulename) {
 		var fullpath = trim(path);
-		if (typeof modulePaths[modulename] !== 'undefined') {
-			fullpath = modulePaths[modulename] +
-
-			// left in for backwards compatibility but shouldn't be needed hereonin
```

Comment:
Yes, since any module using it properly since 0.4.1 will work, so upgrading should be easy (with a very wide semver range of `>=0.4.1 <3`).